### PR TITLE
adding Opportunity mezzanine template generation

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
HTML is currently output to `dest/opportunity/fellowships/index.html`. This is the file we'll need to move into Mezzanine after each `network` CI build finishes.

**Comment by alanmoo** [comment]
Is this ready to review?

**Comment by gvn** [comment]
Not quite.

**Comment by gvn** [comment]
@Pomax @cadecairos Just let me know what additional tokens you'd like added...

File: source/pug/views/opportunity.pug [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+include ../partials/sign-up-box.pug
+
+block centerNav
+  p {{ page.meta_title }}
+
+block content
+  .container
+    .row.py-5
+      .col
+        h1.h1-white {{ page.pagewithcallout.header }}
```
Comment (cadecairos) [inline source/pug/views/opportunity.pug]: we're going to want to have this be `page.landingpage.header

File: source/pug/views/opportunity.pug [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        div {{ page.pagewithcallout.content | safe }}
```
Comment (cadecairos) [inline source/pug/views/opportunity.pug]: `page.landingpage.content`

File: source/pug/views/opportunity.pug [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    .row.py-5
+      .col-md-6
+      +sign-up-box(`{{ page.pagewithcallout.signup.header }}`, `{{ page.pagewithcallout.signup.description | safe }}`)
```
Comment (cadecairos) [inline source/pug/views/opportunity.pug]: `page.landingpage.signup.header` && `page.landingpage.signup.description `

**Comment by gvn** [comment]
Also need to put full URLs in for the external JS/CSS referenced by the template header.

Use: https://network.mofoprod.net/

**Comment by gvn** [comment]
Note: Template is generated to uncommitted path:

`/dest/opportunity/template/index.html`

(Run `npm build` to generate it.)

**Comment by Pomax** [comment]
let's add a domain token and call it `{{ settings.ASSET_DOMAIN }}` and then we *should* be able to tap into that

**Comment by gvn** [comment]
Done!

**Comment by Pomax** [comment]
Trying something "smart", if we change `{{ settings.ASSET_DOMAIN }}` to `{% settings_value "ASSET_DOMAIN" %}` I think we can make this work more easily

**Comment by gvn** [comment]
Token updated!

**Comment by Pomax** [comment]
We apparently need one more thing:
```
<html>
  {% load settings_value %}
  <head>
  ...
```

**Comment by gvn** [comment]
Added!

**Pull request merged**